app/services/email_manager/giveaway_entries_sender.rb

```diff
@@ -19,7 +19,7 @@ def call
         if number_of_tickets % 3 == 0
           entry_status = "You have been entered into this week's giveaway!"
         else
-          entry_status = "You have been entered into this week's giveaway! Only #{number_of_tickets % 3} more tickets until your next entry!"
+          entry_status = "You have been entered into this week's giveaway! Only #{-number_of_tickets % 3} more tickets until your next entry!"
```

Comment:
I think you want 3 - number_of_tickets % 3

Reply:
omg i am so small brain